keel/base/deployment.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keel
+  namespace: spinnaker
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - env:
+        - name: JAVA_OPTS
+          value: -XX:MaxRAMPercentage=50.0
+        image: docker.io/nimak/spinnaker-keel:latest
```

Comment:
the image needs to be pushed to the spinnaker owned repo, so this will have to change.

Reply:
While I don't want to unnecessarily block `kleat` users who want to try deploying `keel`, I don't think I feel comfortable merging this as-is if users deploying this config will be pulling the latest `keel` from a personal, non-Spinnaker image repository. Is `keel` close to being published in the Spinnaker-owned repo? If we do want to move forward with merging this as-is, can we better communicate to users that this image is not yet maintained in the Spinnaker-owned repo and it's a use-at-your-own-risk situation?

Reply:
yeah @maggieneterval I agree and it makes total sense. I am working with the buildcops [here](https://spinnakerteam.slack.com/archives/CKXFWFB53/p1602529528020000) to make this happen. Let's have this PR blocked on building and pushing the image to the official spinnaker registry.